test/metabase/search/fulltext/scoring_test.clj

```diff
@@ -73,18 +72,17 @@
                         :models           search.config/all-models
                         :model-ancestors? false}
                        raw-ctx))]
-      (is (get (search.scoring/scorers search-ctx) ranker-key) "The ranker is enabled")
```

Comment:
What's your reason for removing this?

Reply:
yea, I should have put some context here; I wanted to test that search. There is no feature flag. if we have this the test will fail.